add doc for adding session variable for stale read (#8085)
pingcap/docs-cn: stale-read.md

## 使用方法

TiDB 提供语句级别与会话级别的 Stale Read 使用方式，具体使用方法如下：

- 语句级别：
    - 指定一个精确的时间点（**推荐**）：如需 TiDB 读取一个时间点上保证全局事务记录一致性的数据并且不破坏隔离级别，你可以指定这个时间点对应的时间戳。要使用该方式，请参阅 [`AS OF TIMESTAMP` 语法](/as-of-timestamp.md#语法方式)文档。

>     - 指定时间范围：如需 TiDB 读取在一个时间范围内尽可能新的数据并且不破坏隔离级别，你可以指定一个时间范围。在指定时间范围内，TiDB 会选择一个合适的时间戳，该时间戳能保证所访问的副本上不存在开始于这个时间戳之前且还没有提交的相关事务，即能保证在所访问的可用副本上可执行读取操作而且不会被阻塞。 要使用该方式，请参阅 [`AS OF TIMESTAMP` 语法](/as-of-timestamp.md#语法方式)文档和该文档中 [`TIDB_BOUNDED_STALENESS` 函数](/as-of-timestamp.md#语法方式)部分的介绍。

Reviewer: ```suggestion
    - 指定时间范围：如需 TiDB 读取在一个时间范围内尽可能新的数据并且不破坏隔离级别，你可以指定一个时间范围。在指定时间范围内，TiDB 会选择一个合适的时间戳，该时间戳能保证所访问的副本上不存在开始于这个时间戳之前且还没有提交的相关事务，即能保证在所访问的可用副本上可执行读取操作而且不会被阻塞。要使用该方式，请参阅 [`AS OF TIMESTAMP` 语法](/as-of-timestamp.md#语法方式)文档和该文档中 [`TIDB_BOUNDED_STALENESS` 函数](/as-of-timestamp.md#语法方式)部分的介绍。
```

- 会话级别：

>     - 指定时间范围：在会话级别中，如需 TiDB 在后续的查询中读取一个时间范围内尽可能新的数据并且不破坏隔离级别，你可以通过设置一个 session 变量 `tidb_read_staleness` 来指定一个时间范围。要使用该方式，请参阅 [通过系统变量 `tidb_read_staleness` 读取历史数据](/tidb_read_staleness.md)。

Reviewer: ```suggestion
    - 指定时间范围：在会话级别中，如需 TiDB 在后续的查询中读取一个时间范围内尽可能新的数据并且不破坏隔离级别，你可以通过设置一个 session 变量 `tidb_read_staleness` 来指定一个时间范围。要使用该方式，请参阅[通过系统变量 `tidb_read_staleness` 读取历史数据](/tidb_read_staleness.md)。
```

## 使用方法

TiDB 提供两种语句级别的 Stale Read 使用方式，分别是指定一个精确的时间点和一个时间范围：

- 指定精确时间点（推荐）：如需 TiDB 读取一个时间点上保证全局事务记录一致性的数据并且不破坏隔离级别，你可以指定这个时间点对应的时间戳。要使用该方式，请参阅 [`AS OF TIMESTAMP` 语法](/as-of-timestamp.md#语法方式)文档。
- 指定时间范围：如需 TiDB 读取在一个时间范围内尽可能新的数据并不破坏隔离级别，你可以指定一个时间范围。在指定时间范围内，TiDB 会选择一个合适的时间戳，该时间戳能保证所访问的副本上不存在开始于这个时间戳之前且还没有提交的相关事务，即能保证在所访问的可用副本上可执行读取操作而且不会被阻塞。 要使用该方式，请参阅 [`AS OF TIMESTAMP` 语法](/as-of-timestamp.md#语法方式)文档和该文档中 [`TIDB_BOUNDED_STALENESS` 函数](/as-of-timestamp.md#语法方式)部分的介绍。

> TIDB  提供 Session 级别的 Stale Read 使用方式，指定一个时间范围:
> 
> - 指定时间范围: 如需在 Session 级别让 TiDB 开启后续的查询读取在一个时间范围内尽可能新的数据并不破坏隔离级别，你可以通过设置一个 Session 变量来指定一个时间范围。要使用该方式，请参阅 [通过系统变量 `tidb_read_staleness` 读取历史数据'](/tidb_read_staleness.md)文档。

Reviewer 2: ```suggestion
TiDB 提供会话级别的 Stale Read 使用方式，指定一个时间范围:

- 在会话级别中，如需 TiDB 在后续的查询中读取一个时间范围内尽可能新的数据，且不破坏隔离级别，你可以通过设置一个 session 变量 `tidb_read_staleness` 来指定一个时间范围。有关具体的设置方法，请参阅 [通过系统变量 `tidb_read_staleness` 读取历史数据](/tidb_read_staleness.md)。
```

> TiDB 提供两种语句级别的 Stale Read 使用方式，分别是指定一个精确的时间点和一个时间范围：
> 
> - 指定精确时间点（推荐）：如需 TiDB 读取一个时间点上保证全局事务记录一致性的数据并且不破坏隔离级别，你可以指定这个时间点对应的时间戳。要使用该方式，请参阅 [`AS OF TIMESTAMP` 语法](/as-of-timestamp.md#语法方式)文档。
> - 指定时间范围：如需 TiDB 读取在一个时间范围内尽可能新的数据并不破坏隔离级别，你可以指定一个时间范围。在指定时间范围内，TiDB 会选择一个合适的时间戳，该时间戳能保证所访问的副本上不存在开始于这个时间戳之前且还没有提交的相关事务，即能保证在所访问的可用副本上可执行读取操作而且不会被阻塞。 要使用该方式，请参阅 [`AS OF TIMESTAMP` 语法](/as-of-timestamp.md#语法方式)文档和该文档中 [`TIDB_BOUNDED_STALENESS` 函数](/as-of-timestamp.md#语法方式)部分的介绍。
> 
> TIDB  提供 Session 级别的 Stale Read 使用方式，指定一个时间范围:
> 
> - 指定时间范围: 如需在 Session 级别让 TiDB 开启后续的查询读取在一个时间范围内尽可能新的数据并不破坏隔离级别，你可以通过设置一个 Session 变量来指定一个时间范围。要使用该方式，请参阅 [通过系统变量 `tidb_read_staleness` 读取历史数据'](/tidb_read_staleness.md)文档。

Reviewer 2: ```suggestion
TiDB 提供语句级别与会话级别的 Stale Read 使用方式，具体使用方法如下：

- 语句级别：
    - 指定一个精确的时间点（**推荐**）：如需 TiDB 读取一个时间点上保证全局事务记录一致性的数据并且不破坏隔离级别，你可以指定这个时间点对应的时间戳。要使用该方式，请参阅 [`AS OF TIMESTAMP` 语法](/as-of-timestamp.md#语法方式)文档。
    - 指定时间范围：如需 TiDB 读取在一个时间范围内尽可能新的数据并且不破坏隔离级别，你可以指定一个时间范围。在指定时间范围内，TiDB 会选择一个合适的时间戳，该时间戳能保证所访问的副本上不存在开始于这个时间戳之前且还没有提交的相关事务，即能保证在所访问的可用副本上可执行读取操作而且不会被阻塞。 要使用该方式，请参阅 [`AS OF TIMESTAMP` 语法](/as-of-timestamp.md#语法方式)文档和该文档中 [`TIDB_BOUNDED_STALENESS` 函数](/as-of-timestamp.md#语法方式)部分的介绍。
- 会话级别：
    - 指定时间范围：在会话级别中，如需 TiDB 在后续的查询中读取一个时间范围内尽可能新的数据并且不破坏隔离级别，你可以通过设置一个 session 变量 `tidb_read_staleness` 来指定一个时间范围。要使用该方式，请参阅 [通过系统变量 `tidb_read_staleness` 读取历史数据](/tidb_read_staleness.md)。
```
